README.md

```diff
@@ -1,127 +1,126 @@
+folders as needed. For example, to keep virtualenv files out of your source
+tree, you could add this:
 
-```bash
-$ docker run -i -t \
-$   -p 80:80 \
-$   --name open-oni-dev \
-$   --link mysql:db \
-$   --link solr:solr \
-$   -v /$(pwd)/open-oni/core:/opt/openoni/core \
-$   -v /$(pwd)/data:/opt/openoni/data \
-$   open-oni:dev
+```
+-v /tmp/CachedENV:/opt/openoni/ENV \
 ```
 
-In the above example the `core` folder has been host volume mounted for dynamic development. You can mount additional files / folders as needed. For example:
+Workflow
+---
 
-```
-$ -v /$(pwd)/open-oni/nebraska:/opt/openoni/nebraska \ # for a custom "app"
-$ -v /$(pwd)/settings.py:/opt/openoni/settings.py \ # note: DB_HOST and SOLR_HOST will be overrewritten
-```
+You should be able to develop in the `open-oni` repository folder as normal
+i.e. make feature branches etc.  When ready, make a pull request.  To test a
+pull request simply pull the remote feature branch (again, as normal) and run
+the container including the appropriate volume mount. Using Docker shouldn't
+change your regular development workflow much at all.
 
-Volume mounting is optional. You can always rebuild the image and rerun the container during development. To test modifications to the requirements file re-run the build command. Use the `dev.sh` script to simplify this.
+[`dev.sh`](dev.sh) can be used to stop, rebuild, and restart the dev container in a known
+"reset" state.
 
-**Workflow**
+**Change dependencies**
 
-You should be able to develop in the `open-oni` repository folder as normal i.e. make feature branches etc. For any folder that was volume mounted the changes should be viewable immediately in the browser. When ready, make a pull request. To test a pull request simply pull the remote feature branch (again, as normal) and run the container including the appropriate volume mount (which will typically include `core`). Using Docker shouldn't change your regular development workflow much at all.
+If requirements.pip is changed, you'll need to run the pip install in the container:
 
-For changes that will impact installation a rebuild is required (i.e. changing django versions). In that case `stop` and `rm` open-oni-dev and rebuild with `docker build -t open-oni:dev -f Dockerfile-dev .` and rerun.
+```bash
+docker exec -it open-oni-dev /pip-install.sh
+```
+
+**Load data**
 
-**Note:** do not attempt to volume mount the entire open-oni repository folder. It won't work as the virtualenv environment only exists inside the container, generated at build time. Workaround?
+```bash
+cd data
+wget --recursive --no-host-directories --cut-dirs 1 --reject index.html* --include-directories /data/batches/batch_uuml_thys_ver01/ http://chroniclingamerica.loc.gov/data/batches/batch_uuml_thys_ver01/
+cd ..
+docker exec -it open-oni-dev /load_batch.sh batch_uuml_thys_ver01
+```
 
 **Run tests**
 
 ```bash
-$ docker exec -it open-oni /test.sh
+docker exec -it open-oni-dev /test.sh
 ```
 
 ---
```

Comment:
Do you think it would be worth mentioning everybody's pal `docker exec -it open-oni-dev bash` ?